frontend/app/views/spree/shared/_header.html.erb

```diff
@@ -1,6 +1,6 @@
 <%# leaving #spree-header div for legacy support  %>
 <% cache spree_menu_cache_key('header') do %>
-  <div id="spree-header" data-turbolinks-permanent>
```

Comment:
@MatthewKennedy we need to move this to the cart container like here: https://github.com/spree/spree/pull/10981/files#diff-603b12a43cf4ffa5227d842ffbbd22eb78d98fbbe7f6b309355dc1b573ba8352R39

Reply:
Will fix that.